.bra.toml

```diff
@@ -0,0 +1,22 @@
+# default configuration created by the `mage watch` command.
```

Comment:
`mage watch` was added in https://github.com/grafana/grafana-plugin-sdk-go/pull/563